src/ngraph/runtime/cpu/pass/cpu_memory_optimization.cpp

```diff
@@ -301,6 +301,15 @@ bool runtime::cpu::pass::CPUMemoryOptimization::run_on_function(std::shared_ptr<
                 continue;
             }
 
+            const auto& dtype = slice->get_input_element_type(0);
+            if (runtime::cpu::mkldnn_utils::get_mkldnn_data_type(dtype) ==
+                mkldnn::memory::data_type::data_undef)
+            {
+                NGRAPH_DEBUG << slice->get_input_element_type(0).c_type_string()
+                             << " isn't supported";
```

Comment:
can you please add ", no in place slice" to the debug message for clarity?

Reply:
will do